sbin/pack.sh

```diff
@@ -72,6 +72,7 @@ OSNICK=$($READIES/bin/platform --osnick)
 [[ $OSNICK == ol8 ]]     && OSNICK=rhel8
 [[ $OSNICK == rocky8 ]]  && OSNICK=rhel8
 [[ $OSNICK == rocky9 ]]  && OSNICK=rhel9
+[[ $OSNICK == NotpineForGHA3.20.3 ]]  && OSNICK=alpine3
```

Comment:
It can be removed in favour of changing the `/etc/os-release` file. Check how it is done in RedisTimeSeries in the labs.

Reply:
If I change the ID before this step it can't upload the artifacts to s3 because it use aws-cli

Reply:
in timeseriese currently we not upload artifacts

Reply:
```yaml
      - name: Pack module
        run: |
          git config --global --add safe.directory $GITHUB_WORKSPACE
          mv /etc/os-release.bak /etc/os-release
          make pack BRANCH=$TAG_OR_BRANCH SHOW=1
      - name: Workaround alpine-arm64 GHA issues (one more time)
        shell: sh
        run: |
          sed -i 's/ID=alpine/ID=NotpineForGHA/g' /etc/os-release
 ```

Reply:
but after that in uploading artifacts it should change back to NotpineForGHA